Documentation/Widgets/Index.rst

```diff
@@ -0,0 +1,29 @@
+.. include:: ../Includes.txt
+
+=======
+Widgets
+=======
+
+.. versionadded:: 0.0.1
```

Comment:
As the first release of the extension has this widget included, I think, the version information is not necessary.

Reply:
I removed the "versionadded" information